### PR TITLE
#8 Added RelatedPerson.csv, CSV2RelatedPerson.map and generated instances

### PR DESCRIPTION
**Pull request by heathfrankel**

### CODE REVIEW

File: generated/RelatedPerson-mackay-heather-3.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  "patient": {
+    "reference": "Patient/mackay-elliott"
+  },
+  "relationship": [
+    {
+      "coding": [
+        {
+          "system": "http://terminology.hl7.org/CodeSystem/v3-RoleCode",
+          "code": "MTH",
+          "display": "mother"
```
Comment (StephThaoOng) [inline generated/RelatedPerson-mackay-heather-3.json:63]: for variation and non-biases suggest:
"code": "PRN",
"display": "parent"

File: generated/RelatedPerson-mackay-heather-2.json [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  ],
+  "patient": {
+    "reference": "Patient/mackay-fritz"
+  },
+  "relationship": [
+    {
+      "coding": [
+        {
+          "system": "http://terminology.hl7.org/CodeSystem/v3-RoleCode",
+          "code": "WIFE",
```
Comment (StephThaoOng) [inline generated/RelatedPerson-mackay-heather-2.json:62]: for variation and non-biases suggest:
"code": "SPS",
"display": "spouse"

**Review by StephThaoOng — COMMENTED** [review]
thanks @heathfrankel 
I've reviewed and suggested two changes to support variation and non-biases in the data set. Otherwise everything looks good.

**Comment by heathfrankel** [comment]
Made suggested changes

**Pull request merged**